app/api/api_v1/endpoints/matches.py

```
@@ -115,49 +117,42 @@ def create_match_between_users(
         match = match_service.create_match(
             match_request.user1_id,
-            match_request.user2_id,  
-            match_request.compatibility_score
+            match_request.user2_id,
+            match_request.compatibility_score,
         )
         return MatchOut.from_match(match)
+    except (ValueError, FirebaseError) as e:
+        print(f"{e = }")
+        print(f"{type(e) = }")
     except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)